package.json

```diff
@@ -46,6 +46,7 @@
     "@sentry/ember": "9.40.0",
     "chart.js": "4.5.0",
     "date-fns": "4.1.0",
+    "ember-autofocus-modifier": "^7.0.1",
```

Comment:
we already use `@zestia/ember-auto-focus`. I'm not opposed to switching if that one is better maintained, but that should be done in a dedicated PR